frontend/src/styles/utilities.scss

```diff
@@ -1408,3 +1408,7 @@ $decorations: underline, overline, line-through, no-underline;
 .aspect-video {
     aspect-ratio: 16 / 9;
 }
+
+.h-full {
+    height: '100%';
+}
```

Comment:
I'm surprised we don't already have this... it should probably live alongside the code that defines the other `h` variables.

Is this and the button change necessary? There are no screenshot changes so we're either not covering a button state that we're changing or it isn't actually changing the buttons

Reply:
I see a lot of usage of this class but no matching definition for it. 

The problem I was fixing is that the disabled button is shorter than the non-disabled button. I suppose that since we're not going to disable the clear button, there's no need for this change.

Reply:
Yep, let's yank it from here...

I can't see the definition either... this is the fun of our almost using tailwind generation of CSS - let's pray for #19051

Reply:
<img width="458" alt="Screenshot 2024-01-06 at 15 29 17" src="https://github.com/PostHog/posthog/assets/984817/654b24c6-686a-44e2-800c-8425e7faf657">

Ah, figured it out while waiting for the kids swimming lesson to finish... we slice h and w off height and width and build a few rules including h-full and w-full